mmv1/third_party/terraform/services/netapp/resource_netapp_backup_vault_test.go

```diff
@@ -67,6 +67,49 @@ resource "google_netapp_backup_vault" "test_backup_vault" {
 `, context)
 }
 
+func TestAccNetappBackupVault_Kms(t *testing.T) {
+	location := "us-east4"
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      location,
+		"kms_key_name":  acctest.BootstrapKMSKeyWithPurposeInLocationAndName(t, "ENCRYPT_DECRYPT", location, "tf-test-netapp-bv-key").CryptoKey.Name,
+	}
+
```

Comment:
To add the necessary roles to the service account you can use `acctest.BootstrapIamMembers()` ([here's an example](https://github.com/GoogleCloudPlatform/magic-modules/blob/13217f35ae175e5aeaf6ad02ca68488b52d023a6/mmv1/third_party/terraform/services/cloudfunctions2/resource_cloudfunctions2_function_test.go.tmpl#L196)). Based on the list of permissions required you may need to add a couple of roles (`roles/cloudkms.cryptoKeyEncrypterDecrypter` and `roles/cloudkms.viewer` maybe?)